packages/common/src/SVGZoomWidget.ts

```diff
@@ -179,9 +179,17 @@ export class SVGZoomWidget extends SVGWidget {
         }
     }
 
-    zoomToFit(transitionDuration?) {
+    zoomToFit(transitionDuration?, bboxScale?: number) {
         if (this._renderElement) {
             const bbox = this._renderElement.node().getBBox();
+            if (bboxScale) {
+                const widthDiff = (bbox.width * bboxScale) - bbox.width;
+                const heightDiff = (bbox.height * bboxScale) - bbox.height;
```

Comment:
Rather than a "scale" would be better to have an absolute padding (i.e. 16px not 16%)

Reply:
Scaling it with a multiplier is a quick win, but supporting pixel padding seems like a bigger change... because I think the zoom would have to be considered?

Reply:
Would it not be a matter of subtracting 16px from  bbox.width / height?

Reply:
You're right, I'm not sure why the scaling seemed more reliable, but px can be used here.